docs/developer/extending/webhooks/synchronous-events/shipping.mdx

```diff
@@ -217,8 +217,12 @@ Saleor Apps can subscribe to the synchronous webhooks:
 - `ORDER_FILTER_SHIPPING_METHODS` is called whenever shipping methods are listed for an order
 
 :::warning
-Due to circular reference, subscription queries should not contain `checkout.shippingMethods` or `order.shippingMethods` fields.
-:::
+Due to circular reference, subscription queries should not contain fields:
+
+- `checkout.shippingMethods`,
+- `checkout.deliveryMethods`,
+- `order.shippingMethods`.
+  :::
```

Comment:
Looks like it broke UI 
![CleanShot 2024-03-20 at 10 56 25@2x](https://github.com/saleor/saleor-docs/assets/9116238/39dc8fbf-35c6-45b3-a161-5a38cb43c6c8)
https://saleor-docs-git-updateshippingeventswarning-saleorcommerce.vercel.app/docs/3.x/developer/extending/webhooks/synchronous-events/shipping#filter-shipping-methods